src/bci_build/package/__init__.py

```diff
@@ -1358,6 +1358,13 @@ def reference(self) -> str:
         return f"{self.registry}/{self._registry_prefix}/bci-{self.name}:{self.version_label}"
 
 
+@dataclass
+class OsFIPSContainer(OsContainer):
```

Comment:
I would prefer a different name here, e.g.
```suggestion
class LTSSContainer(OsContainer):
```